.github/maintainers_guide.md

```diff
@@ -110,11 +110,12 @@ Place `$HOME/.m2/settings.xml` with your Sonatype account information.
 
 #### Operations
 
+* From the upstream repository
 * Preparation
-  * `git pull --rebase slackapi master`
+  * `git switch main && git pull origin main`
   * Make sure there are no build failures at https://github.com/slackapi/java-slack-sdk/actions
 * Set a new version
-  * If you don't have `gnu-sed`, run `brew install gnu-sed` first
+  * If you don't have `gnu-sed`, run `brew install gnu-sed` & `brew install gnupg` first
```

Comment:
Just installing GnuPG is not enough. Before making the release, there are a few steps to complete:
- Set up your key: https://central.sonatype.org/publish/requirements/gpg/
- Make sure if your account has the permission to make releases [under com.slack groupId](https://central.sonatype.com/publishing/com.slack/users) (note that this page requires login as the account)
- Update your $HOME/.m2/settings.xml 
  - generate user token: https://central.sonatype.org/publish/generate-token/
  - set the user token id/password: https://central.sonatype.org/publish/publish-manual/#signing-components